monai/apps/reconstruction/networks/nets/utils.py

```diff
@@ -135,6 +161,109 @@ def complex_normalize(x: Tensor) -> Sequence:  # type: ignore
         x = x.view(b, c, h, w, d)
         return (x - mean) / std, mean, std  # type: ignore
 
+    else:
+        raise ValueError(f"only 2D (B,C,H,W) and 3D (B,C,H,W,D) data are supported but x has shape {x.shape}")
+
+
+def divisible_pad_t(
+    x: Tensor, k: int = 16
+) -> Tuple[Tensor, Tuple[Tuple[int, int], Tuple[int, int], Tuple[int, int], int, int, int]]:  # type: ignore
+    """
+    Pad input to feed into the network (torch script compatible)
+
+    Args:
```

Comment:
*New function*

Two reasons behind adding this util:
1- Monai's `DivisiblePad` and `SpatialPad` are not jit script compatible
2- There is a difference between those two and `torch.nn.functional.pad` . Don't know what's the difference exactly but somehow the latter works better in image reconstruction tasks.